src/GraphQL.Tests/Execution/ExecutionNodeTests.cs

```diff
@@ -0,0 +1,91 @@
+using System.Collections.Generic;
+using System.Linq;
+using GraphQL.Execution;
+using GraphQL.Types;
+using Xunit;
+
+using AST = GraphQL.Language.AST;
+
+namespace GraphQL.Tests.Execution
+{
+    public class ExecutionNodeTests
+    {
+        [Fact]
+        public void Path_Alias()
+        {
+            var objectGraphType = new AliasedFieldTestObject();
+
+            var node = new ValueExecutionNode(
+                new RootExecutionNode(objectGraphType),
+                new StringGraphType(),
+                new AST.Field(new AST.NameNode("alias"), new AST.NameNode("name")),
+                objectGraphType.GetField("value"),
+                indexInParentNode: null);
+
+            var path = node.Path.ToList();
+            string pathSegment = Assert.Single(path) as string;
```

Comment:
This project uses `Shouldly` for assertions.  Would you mind updating these tests to use that?

Reply:
fixed